lua/simple_policy.lua

```diff
@@ -0,0 +1,89 @@
+local ffi = require("ffi")
```

Comment:
It'd be better if you merge file `lua/simple_policy.lua` into file `lua/policy.lua` since we'll later do a full rewrite of the policy we'll make public.